lib/rubocop/cop/lint/interpolation_check.rb

```diff
@@ -21,11 +21,10 @@ class InterpolationCheck < Cop
               'Use double quoted strings if you need interpolation.'
 
         def on_str(node)
-          return if heredoc?(node)
-
           parent = node.parent
           return if parent && (parent.dstr_type? || parent.regexp_type?)
           return unless /(?<!\\)#\{.*\}/.match?(node.source.scrub)
```

Comment:
AFAIK, `parser` only handles valid utf-8 sequences, so `scrub` should not be needed at all here, or am I missing something?